doc/Language/control.pod6

```diff
@@ -776,7 +776,7 @@ L<supply|/language/concurrency#index-entry-supply_(on-demand)>:
     # received Rat (0.5)
 
 =head1 X<given|control flow, given>
-X<|switch (given)>
+X<|switch (given);case statements (given)>
```

Comment:
It's probably better if you add it in another X clause, as in X<|case statements (given)>